Assets/Mirror/Runtime/NetworkIdentity.cs

```diff
@@ -186,12 +186,12 @@ internal set
         // used when adding players
         internal void SetClientOwner(NetworkConnection conn)
         {
-            if (clientAuthorityOwner != null)
+            if (connectionToClient != null && conn != connectionToClient)
             {
-                Debug.LogError("SetClientOwner m_ClientAuthorityOwner already set!");
+                Debug.LogError($"Object {this} netId={netId} already has an owner", this);
```

Comment:
```suggestion
                Debug.LogError($"Object {this} netId={netId} already has an owner");
```
Don't need that `this` there :)